EIPS/eip-3540.md

```diff
@@ -262,43 +246,20 @@ The choice of `MAGIC` guarantees that none of the contracts existing on the chai
 
 ## Test Cases
 
-### Contract creation
-
-All cases should be checked for creation transaction, `CREATE` and `CREATE2`.
+### Container validation rules
```

Comment:
I would just remove Test Cases section and focus on providing state tests instead.